HTML/ex5/ex5.js

```diff
@@ -0,0 +1,145 @@
+let sortOption = 'ascending'; // Default sort option
+let currentIndex = 0; // Keep track of the current index globally
+let sortedMovies = []; // Store sorted movies globally
+const moviesPerPage = 3; // Number of movies to show per load
+
+document.addEventListener("DOMContentLoaded", async function () {
+    const apiKey = '262e0edd'; // Replace with your OMDb API key
+    const url = `http://www.omdbapi.com/?apikey=${apiKey}&s=${encodeURIComponent('blade')}`;
```

Comment:
`apiKey` and `url` is declared multiple times. Can be declared once and passed to required functions.